<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminDialog</class>
 <widget class="QDialog" name="AdminDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
  font-family: 'Noto Sans KR';
  font-size: 13px;
  color: #1D1D1F;
  background-color: #F5F5F7;
}
QLineEdit, QSpinBox, QTextEdit, QComboBox {
  background: #FFFFFF;
  border: 1px solid #D2D2D7;
  border-radius: 8px;
  padding: 6px 8px;
}
QPushButton {
  background: #0071E3;
  color: #FFFFFF;
  border: none;
  border-radius: 8px;
  padding: 8px 12px;
}
QPushButton:disabled {
  background: #C7C7CC;
  color: #FFFFFF;
}
QGroupBox {
  border: 1px solid #E5E5EA;
  border-radius: 12px;
  margin-top: 8px;
  background: #FFFFFF;
}
QGroupBox::title {
  subcontrol-origin: margin;
  left: 12px;
  padding: 0 4px;
  font-weight: 600;
}</string>
  </property>
  <layout class="QVBoxLayout" name="rootLayout">
   <property name="leftMargin">
    <number>16</number>
   </property>
   <property name="topMargin">
    <number>16</number>
   </property>
   <property name="rightMargin">
    <number>16</number>
   </property>
   <property name="bottomMargin">
    <number>16</number>
   </property>
   <property name="spacing">
    <number>12</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="monthRow">
     <item>
      <widget class="QLabel" name="lblMonthTitle">
       <property name="text">
        <string>월 선택</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbMonth"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="grpStats">
     <property name="title">
      <string>월 통계</string>
     </property>
     <layout class="QGridLayout" name="statsGrid">
      <property name="horizontalSpacing">
       <number>12</number>
      </property>
      <property name="verticalSpacing">
       <number>8</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lblStatTotalVisitsTitle">
        <property name="text">
         <string>이번달 입력횟수</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lblStatTotalVisits">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblStatRewardGrantedTitle">
        <property name="text">
         <string>포인트 지급 횟수</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="lblStatRewardGranted">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblStatPointUsedTitle">
        <property name="text">
         <string>포인트 사용 횟수</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="lblStatPointUsed">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblStatLoyalRatioTitle">
        <property name="text">
         <string>단골 비율</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="lblStatLoyalRatio">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblTop5Title">
        <property name="text">
         <string>Top5</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QListWidget" name="listTop5Customers"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lblStatMoMCompareTitle">
        <property name="text">
         <string>전월 대비</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="lblStatMoMCompare">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpManualGrant">
     <property name="title">
      <string>포인트 수동 지급</string>
     </property>
     <layout class="QGridLayout" name="manualGrid">
      <property name="horizontalSpacing">
       <number>12</number>
      </property>
      <property name="verticalSpacing">
       <number>8</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lblManualPhoneTitle">
        <property name="text">
         <string>전화번호</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="leManualPhone">
        <property name="placeholderText">
         <string>전화번호(숫자만)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblManualPointsTitle">
        <property name="text">
         <string>포인트</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spnManualPoints">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblManualReasonTitle">
        <property name="text">
         <string>사유</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="leManualReason">
        <property name="placeholderText">
         <string>지급 사유(필수)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QPushButton" name="btnManualGrant">
        <property name="text">
         <string>포인트 수동 지급</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  <item>
   <widget class="QGroupBox" name="grpDeleteCustomer">
    <property name="title">
     <string>고객 삭제</string>
    </property>
    <layout class="QGridLayout" name="deleteGrid">
     <property name="horizontalSpacing">
      <number>12</number>
     </property>
     <property name="verticalSpacing">
      <number>8</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lblDeletePhoneTitle">
       <property name="text">
        <string>전화번호</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="leDeletePhone">
       <property name="placeholderText">
        <string>전화번호(숫자만)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QPushButton" name="btnDeleteCustomer">
       <property name="text">
        <string>고객 삭제</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </item>
   <item>
    <widget class="QPushButton" name="btnCloseAdmin">
     <property name="text">
      <string>닫기</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
